src/views/TypingGame.vue

```diff
@@ -26,210 +26,183 @@
+      ['is', 'seems', 'becomes', 'appears', 'feels', 'looks', 'sounds', 'smells', 'tastes', 'remains', 'grows', 'stays', 'turns', 'sits', 'stands', 'falls', 'fades', 'shines', 'feels', 'moves', 'acts', 'sounds', 'becomes', 'feels', 'seems', 'grows', 'appears', 'looks', 'remains', 'seems', 'turns', 'stays', 'feels', 'appears', 'sounds', 'turns'],
+      ['and', 'but', 'or'],
     ];
+    const wordType = {
+      article: 0,
+      person: 1,
+      verb: 2,
+      adverb: 3,
+      objectNoun: 4,
+      subject: 5,
+      preposition: 6,
+      adjective: 7,
+      linkingVerb: 8,
+      conjunction: 9,
+    }
+  
+    const sentence = [
+      [wordType.article, wordType.subject, wordType.adverb, wordType.verb, wordType.preposition, wordType.article, wordType.adjective, wordType.objectNoun],
+      [wordType.person, wordType.verb, wordType.adverb, wordType.conjunction, wordType.linkingVerb, wordType.article, wordType.adjective, wordType.objectNoun],
+    ]
+    const space = " ";
+    const period = ".";
+    const texts = [];
+    const roundsToPlay = 3;
+    const curRound = ref(0);
 
-    const gerunds = [
-      "running", "jumping", "swimming", "driving", "reading", "writing", "singing", "dancing", "cooking", "playing",
-      "studying", "teaching", "painting", "drawing", "building", "fixing", "throwing", "catching", "flying", "climbing"
-    ];
-    const items = [
-      "chair", "table", "lamp", "sofa", "bed", "desk", "cabinet", "television", "rug", "shelf",
-      "couch", "dresser", "stove", "microwave", "refrigerator", "toaster", "blender", "coffee maker", "oven", "bookshelf"
-    ];
+    
+    function generateSentence() {
+      return appendWord();
+    }
 
-    const tenseWords = [
-      "present",    // e.g., "run"
-      "past",       // e.g., "ran"
-      "future",     // e.g., "will run"
-      "presentContinuous", // e.g., "running"
-      "pastContinuous",    // e.g., "was running"
-      "presentPerfect",    // e.g., "have run"
-      "pastPerfect",       // e.g., "had run"
-      "futureContinuous",  // e.g., "will be running"
-      "futurePerfect",     // e.g., "will have run"
-      "conditional"        // e.g., "would run"
-    ];
-    const tensePostfixes = [
-      "ing",        // Present Continuous, Gerunds
-      "ed",         // Past Simple
-      "s",          // Present Simple (third person singular)
-      "en",         // Past Participle (e.g., "written")
-      "will",       // Future Simple
-      "was/were",   // Past Continuous
-      "have/has",   // Present Perfect
-      "had",        // Past Perfect
-      "be",         // Future Continuous
-      "would"       // Conditional
-    ];
+    function appendWord() {
+      let firstWord = "";
+      let i = Math.floor(Math.random() * sentence.length);
```

Comment:
What is this for?

Why use random?

Reply:
We have an array of sentences, this `i` index randomly chooses the sentence.